### PR TITLE
Plat UI 318 - introduce ability to control versions of axe, vnu and chromedriver

### PR DESCRIPTION
**Pull request by vivrichards600**

### CODE REVIEW

File: docker/files/accessibility_tools/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+control the versions of chromedriver, axe and vnu which are used within the accessibility assessment.
+ 
+
+## .npmrc
+This file specifies an npm config property which detects compatibility of the currently installed chromedriver
+version before installing a new version. 
+
+Further information on this npm config property can be found here https://www.npmjs.com/package/chromedriver
+
+### Updating chromedriver
```
Comment (ejayaraman) [inline docker/files/accessibility_tools/README.md:13]: Isn't updating Chromedriver documented in README.md? I am not sure if it fits in this part of README.

File: docker/Dockerfile [hunk 2]
```diff
@@ -16,17 +16,21 @@ USER root
...
       && sudo chmod -R 755 ${HOME}/app \
       && sudo mkdir ${HOME}/output \
       && sudo chmod 755 ${HOME}/output \
       && sudo chown -R seluser:seluser ${HOME}/output \
       && sudo mkdir $NVM_DIR \
       && sudo chown seluser:seluser $NVM_DIR \
-      && sudo echo -e '#!/bin/bash\njava -jar ${HOME}/.nvm/versions/node/v${NODE_VERSION}/lib/node_modules/vnu-jar/build/dist/vnu.jar "$@"' > /usr/bin/vnu \
-      && chmod +x /usr/bin/vnu
+      && sudo echo -e '#!/bin/bash\njava -jar ${HOME}/accessibility_tools/node_modules/vnu-jar/build/dist/vnu.jar "$@"' > /usr/bin/vnu \
+      && sudo echo -e '#!/bin/bash\n cd ${HOME}/accessibility_tools \n npx axe "$@"' > /usr/bin/axe \
```
Comment (ejayaraman) [inline docker/Dockerfile:31]: If there is a way to add an in line Docker comment, we should add a line explaining what is being done here.

**Review by ejayaraman — CHANGES_REQUESTED** [review]

File: docker/files/accessibility_tools/README.md (hunk 1 above)
Comment (vivrichards600) [reply docker/files/accessibility_tools/README.md]: Removed in latest commit.

File: docker/Dockerfile (hunk 2 above)
Comment (vivrichards600) [reply docker/Dockerfile]: Comment added as discussed which is within the latest commit.

**Review by ejayaraman — APPROVED** [review]

**Pull request merged**